src/Feature/Checkout/tests/Controllers/OrderControllerTests.cs

```diff
@@ -0,0 +1,172 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Feature.Checkout.Tests.Controllers
+{
+    using System;
+    using System.Web;
+    using System.Web.Mvc;
+    using System.Web.Routing;
+
+    using Checkout.Controllers;
+
+    using Foundation.Commerce.Models;
+    using Foundation.Commerce.Models.Checkout;
+    using Foundation.Commerce.Services.Order;
+    using Foundation.Extensions.Models;
+
+    using Models.Requests;
+
+    using NSubstitute;
+    using NSubstitute.ExceptionExtensions;
+
+    using Xunit;
+
+    // TODO: Replace models with new one
```

Comment:
Did you create a task in tracking system for that?

Reply:
not yet, I will